codegen/src/main/kotlin/tools/samt/codegen/http/HttpTransport.kt

```diff
@@ -66,10 +67,19 @@ object HttpTransportConfigurationParser : TransportConfigurationParser {
                             "PATCH" -> HttpTransportConfiguration.HttpMethod.Patch
                             else -> {
                                 params.reportError("Invalid http method '$method'", operationConfig)
-                                return@mapNotNull null
+                                return@forEach
                             }
                         }
 
+                        // check for duplicate path+method combinations
+                        val duplicate = list.find { op ->
+                            op.path == path && op.method == methodEnum
```

Comment:
What about the same operation across services? A provider could implement two interfaces (e.g. `v1.MyService` and `v2.MyService` and have dedicated base routes (e.g. `/v1` and `/v2`). If he now forgets this baseRoute, the same runtime path could exist across services, which is not caught by this approach

Reply:
This has been implemented partly. If there is an explicit configuration that is in conflict with another one (wether in the same service or not) it will throw an error. However, if no configuration of any kind is given, we will not catch the error and emit faulty code.

One way of fixing this would be to change the default basePath of services to their name, instead of just the empty string. That way every service/operation would have a unique path.

What are your thoughts on this approach?

Reply:
I thought about it and believe that the basePath could realistically default, but if you only want to implement a single Service this is dubious.

Maybe it's better if we pass the record type into the default implementation and verify it there?

Reply:
I'm not sure what you mean by passing the record type into the default implementation?

Reply:
Instead of calling `default` we could pass a `defaultFor(provider: ProviderType)` or something of that nature. We could also instead remove the `default` and just make the object node parameter nullable